ParisBNF/abb/BNFabb41.xml

```diff
@@ -0,0 +1,387 @@
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsThreeYouths1"/>
+                           <note/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2.9">
+                           <locus from="430" to="431"/>
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsThreeYouths2"/>
+                           <note/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2.10">
+                           <locus from="431" to="436"/>
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsThreeYouths3"/>
+                           <note/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2.11">
+                           <locus from="436" to="441"/>
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsHabakkuk"/>
+                           <note/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2.12">
+                           <locus from="441" to="444"/>
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsIsaiah"/>
+                           <note/>
+                        </msItem>
+                        <msItem xml:id="ms_i1.2.13">
+                           <locus from="444" to="445"/>
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsMary"/>
+                           <note/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2.14">
+                           <locus from="445" to="447"/>
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsZachariah"/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2.15">
+                           <locus target="#447"/>
+                           <title type="complete" ref="LIT6368PsaltComm#CanticlesProphetsSimeon"/>
+                           <note/>
+                        </msItem>
+                     </msItem>
+
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="447" to="471"/>
+                        <title type="complete" ref="LIT6368PsaltComm#SongSongs"/>
+                        <textLang mainLang="am" otherLangs="gez"/>
+                        <incipit>
+                        <locus target="447"/>
+                         <lb n="1"/><hi rend="rubric">መኃልየ፡ መኃልይ፡ ዝውእቱ፡ ዘሰሎሞን። እግዚአብሔርና፡
+                         እስራኤል፡ የተመሰገኑበት፡ <seg rend="above">መጽሐፍ፡ ይኽ፡ ነው።</seg></hi>
+                   <lb n="2"/>፩ይስዕመኒ፡ በስዕመተ፡ አፉሁ፨ ወትሮ፡ በሚናገር፡ አፉ፡ ይናገረኛል፨
+                     <lb n="3"/>አዳም፡ አጥባትኪ፡ እም፡ ወይን፨ ኹለቱ፡ ጽላትሽ፡ ከወይን፡ ይሻላሉ፡ ሲል፡ ይናገረኛል፨
+                      </incipit>
+                      <note>More text is printed in <bibl><ptr target="bm:ContiRossini1914Notice"/>
```

Comment:
I understand the reasons behind this statement :) but this is against our policy not to lose any information from the catalogues, which I think is very important for the project. Could you please either add the text or possibly collect this missing text in an issue for someone (but who?) to add later. It might be easier back in the office with Abbyy. Are there other records in which text provided by the catalogues has not been added?

Reply:
No problem; usually I do include everything. :)